Potion.py

```diff
@@ -0,0 +1,63 @@
+class Potion:
+    def __init__(self, name, effect, potency):
+        self.name = name
+        self.effect = effect
+        self.potency = potency
+
```

Comment:
Hmm...something about this line I don't like too much. I was thinking something like +10 health or something. No change for now but I will think of something eventually.